jib-core/src/main/java/com/google/cloud/tools/jib/registry/ManifestPusher.java

```diff
@@ -53,6 +55,35 @@ public BlobHttpContent getContent() {
     return Collections.emptyList();
   }
 
+  @Override
+  public Void handleHttpResponseException(HttpResponseException httpResponseException)
```

Comment:
Fixing #601 too in this PR? Should we close #932? Looking at the PR title and the description, I assume you did not really intend to do it here.

Reply:
It includes — it should get tossed once #932 is merged.